### ISSUE TITLE
Launchpad does not build

### ISSUE THREAD

**Issue by parkmino** [jcorporation/myMPD#217]
myMPD version: 6.1.0

**Describe the bug**

Launchpad does not build with following message.

```
 * Source Package: mympd
 * Version: 6.1.0-1ppa1
 * Architecture: amd64
 * Archive: ~parkmino/ubuntu/mympd
 * Component: main
 * State: Chroot problem
 * Duration: 2 minutes
 * Build Log: https://launchpad.net/~parkmino/+archive/ubuntu/mympd/+build/18619809/+files/buildlog_ubuntu-bionic-amd64.mympd_6.1.0-1ppa1_BUILDING.txt.gz
 * Builder: https://launchpad.net/builders/lcy01-amd64-029
 * Source: not available

https://launchpad.net/~parkmino/+archive/ubuntu/mympd/+build/18619809
RUN: /usr/share/launchpad-buildd/bin/builder-prep 
Kernel version: Linux lcy01-amd64-029 4.4.0-171-generic #200-Ubuntu SMP Tue Dec 3 11:04:55 UTC 2019 x86_64
Buildd toolchain package versions: launchpad-buildd_186 python-lpbuildd_186 sbuild_0.67.0-2ubuntu7.1 bzr-builder_0.7.3+bzr174~ppa13~ubuntu14.10.1 bzr_2.7.0-2ubuntu3.1 git-build-recipe_0.3.6~git201906051340.ff11471~ubuntu16.04.1 git_1:2.7.4-0ubuntu1.7 dpkg-dev_1.18.4ubuntu1.6 python-debian_0.1.27ubuntu2.
Syncing the system clock with the buildd NTP service...
27 Jan 22:33:10 ntpdate[1929]: adjust time server 10.211.37.1 offset -0.014038 sec
RUN: /usr/share/launchpad-buildd/bin/in-target unpack-chroot --backend=chroot --series=bionic --arch=amd64 PACKAGEBUILD-18619809 --image-type chroot /home/buildd/filecache-default/540947c8a8a5222e9f34150afed945f9f01ccf43
Creating target for build PACKAGEBUILD-18619809
RUN: /usr/share/launchpad-buildd/bin/in-target mount-chroot --backend=chroot --series=bionic --arch=amd64 PACKAGEBUILD-18619809
Starting target for build PACKAGEBUILD-18619809
RUN: /usr/share/launchpad-buildd/bin/in-target override-sources-list --backend=chroot --series=bionic --arch=amd64 PACKAGEBUILD-18619809 'deb http://ppa.launchpad.net/parkmino/mympd/ubuntu bionic main' 'deb http://ftpmaster.internal/ubuntu bionic main restricted universe multiverse' 'deb http://ftpmaster.internal/ubuntu bionic-security main restricted universe multiverse' 'deb http://ftpmaster.internal/ubuntu bionic-updates main restricted universe multiverse'
Overriding sources.list in build-PACKAGEBUILD-18619809
RUN: /usr/share/launchpad-buildd/bin/in-target update-debian-chroot --backend=chroot --series=bionic --arch=amd64 PACKAGEBUILD-18619809
Updating target for build PACKAGEBUILD-18619809
Hit:1 http://ftpmaster.internal/ubuntu bionic InRelease
Get:2 http://ppa.launchpad.net/parkmino/mympd/ubuntu bionic InRelease [15.9 kB]
Get:3 http://ftpmaster.internal/ubuntu bionic-security InRelease [88.7 kB]
Get:4 http://ftpmaster.internal/ubuntu bionic-updates InRelease [88.7 kB]
Err:2 http://ppa.launchpad.net/parkmino/mympd/ubuntu bionic InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY E2B2505901990533
Get:5 http://ftpmaster.internal/ubuntu bionic-security/main amd64 Packages [611 kB]
Get:6 http://ftpmaster.internal/ubuntu bionic-security/main Translation-en [200 kB]
Get:7 http://ftpmaster.internal/ubuntu bionic-security/restricted amd64 Packages [18.4 kB]
Get:8 http://ftpmaster.internal/ubuntu bionic-security/restricted Translation-en [5408 B]
Get:9 http://ftpmaster.internal/ubuntu bionic-security/universe amd64 Packages [634 kB]
Get:10 http://ftpmaster.internal/ubuntu bionic-security/universe Translation-en [213 kB]
Get:11 http://ftpmaster.internal/ubuntu bionic-security/multiverse amd64 Packages [6116 B]
Get:12 http://ftpmaster.internal/ubuntu bionic-security/multiverse Translation-en [2600 B]
Get:13 http://ftpmaster.internal/ubuntu bionic-updates/main amd64 Packages [839 kB]
Get:14 http://ftpmaster.internal/ubuntu bionic-updates/main Translation-en [294 kB]
Get:15 http://ftpmaster.internal/ubuntu bionic-updates/restricted amd64 Packages [28.5 kB]
Get:16 http://ftpmaster.internal/ubuntu bionic-updates/restricted Translation-en [7568 B]
Get:17 http://ftpmaster.internal/ubuntu bionic-updates/universe amd64 Packages [1045 kB]
Get:18 http://ftpmaster.internal/ubuntu bionic-updates/universe Translation-en [323 kB]
Get:19 http://ftpmaster.internal/ubuntu bionic-updates/multiverse amd64 Packages [9516 B]
Get:20 http://ftpmaster.internal/ubuntu bionic-updates/multiverse Translation-en [4540 B]
Reading package lists...
W: GPG error: http://ppa.launchpad.net/parkmino/mympd/ubuntu bionic InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY E2B2505901990533
E: The repository 'http://ppa.launchpad.net/parkmino/mympd/ubuntu bionic InRelease' is not signed.
Waiting 15 seconds and trying again ...
Hit:2 http://ftpmaster.internal/ubuntu bionic InRelease
Get:1 http://ppa.launchpad.net/parkmino/mympd/ubuntu bionic InRelease [15.9 kB]
Hit:3 http://ftpmaster.internal/ubuntu bionic-security InRelease
Hit:4 http://ftpmaster.internal/ubuntu bionic-updates InRelease
Err:1 http://ppa.launchpad.net/parkmino/mympd/ubuntu bionic InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY E2B2505901990533
Reading package lists...
W: GPG error: http://ppa.launchpad.net/parkmino/mympd/ubuntu bionic InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY E2B2505901990533
E: The repository 'http://ppa.launchpad.net/parkmino/mympd/ubuntu bionic InRelease' is not signed.
Traceback (most recent call last):
  File "/usr/share/launchpad-buildd/bin/in-target", line 27, in <module>
    sys.exit(main())
  File "/usr/share/launchpad-buildd/bin/in-target", line 23, in main
    return args.operation.run()
  File "/usr/lib/python2.7/dist-packages/lpbuildd/target/apt.py", line 90, in run
    self.backend.run(update_args, env=env, stdin=devnull)
  File "/usr/lib/python2.7/dist-packages/lpbuildd/target/chroot.py", line 88, in run
    subprocess.check_call(cmd, **kwargs)
  File "/usr/lib/python2.7/subprocess.py", line 541, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['sudo', '/usr/sbin/chroot', '/home/buildd/build-PACKAGEBUILD-18619809/chroot-autobuild', 'linux64', 'env', 'LANG=C', 'DEBIAN_FRONTEND=noninteractive', 'TTY=unknown', '/usr/bin/apt-get', '-uy', 'update']' returned non-zero exit status 100
RUN: /usr/share/launchpad-buildd/bin/in-target scan-for-processes --backend=chroot --series=bionic --arch=amd64 PACKAGEBUILD-18619809
Scanning for processes to kill in build PACKAGEBUILD-18619809
```

**Comment by jcorporation** [comment]
It seems to be a problem of launchpad in general or they changed anything:
```
Err:2 http://ppa.launchpad.net/parkmino/mympd/ubuntu bionic InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY E2B2505901990533
```

**Comment by parkmino** [comment]
Successfully build at 2nd trial, my password might have been wrong with the signature.
